tailwind.config.mjs

```diff
@@ -26,12 +26,8 @@ export default {
         'slide-in-from-right': 'slide-in-from-right 0.3s forwards ease-in-out',
       },
       borderRadius: {
-        // Tailwind provides a default set of border-radius utility styles
-        // in rem units. Add some values for places where border radius needs
-        // to be a fixed size and not scale with changes to root font size
-        // example: bucket bar indicator buttons
-        'px-sm': '2px',
-        px: '4px',
```

Comment:
These were unused after this consolidation pass, which is good. The fewer entries in this config, the healthier.